ProgrammingGuide/Kokkos-and-Virtual-Functions.md

```diff
@@ -34,13 +34,13 @@ At a glance this should be fine, we've made a device instance of a class, copied
 
 Virtual functions allow a program to handle Derived classes through a pointer to their Base class and have things work as they should. To make this work, the compiler needs some way to identify whether a pointer which is nominally to a Base class really is a pointer to the Base, or whether it's really a pointer to any Derived class. This happens through VPointers and VTables. For every class with virtual functions, there is one VTable shared among all instances, this table contains function pointers for all the virtual functions the class implements. 
 
-![VTable](VirtualFunctions-VTables.png)
+![VTable](https://raw.githubusercontent.com/wiki/kokkos/kokkos/UseCases/VirtualFunctions-VTables.png)
```

Comment:
Did you check if `UseCases/VirtualFunctions-VTables.png` works?

Reply:
I checked that the images get properly displayed using my fork of [ProgrammingGuide repo](https://github.com/ldh4/ProgrammingGuide/wiki/VirtualFunctions)
I am not sure if that link is accessible since it seems to be labeled private.

Reply:
`UseCases/VirtualFunctions-VTables.png` doesn't work for me but this does (testing with the preview tab editing https://github.com/kokkos/kokkos/wiki/Kokkos-and-Virtual-Functions directly).